packages/datastore/src/query.js

```diff
@@ -200,7 +200,7 @@ Query.prototype.select = function(fieldNames) {
 /**
  * Set a starting cursor to a query.
  *
- * @resource [Query Cursors]{@link https://cloud.google.com/datastore/docs/concepts/queries#Datastore_Query_cursors}
+ * @resource [Query Cursors]{@link https://cloud.google.com/datastore/docs/concepts/queries#cursors_limits_and_offsets}
```

Comment:
We can probably link to the same place as the following two links:

https://cloud.google.com/datastore/docs/concepts/queries#datastore-limit-nodejs